test/4.x/app.js

```diff
@@ -128,17 +129,16 @@ app.get('/escape', function(req, res) {
 });
 
 app.get('/locals', function(req, res) {
-  res.locals.person = 'Alan';
   res.render('locals', {
-    layout: false,
+    layout: 'layout_locals',
```

Comment:
Why is this `false` changed? The whole point of false here is to test not using a layout. If there is another test that tests for using false, then this is fine, if there is not we need the two tests.